pkgs/applications/kde/akonadi/default.nix

```diff
@@ -2,8 +2,8 @@
   mkDerivation, lib, kdepimTeam,
   extra-cmake-modules, shared-mime-info, qtbase, accounts-qt,
   boost, kaccounts-integration, kcompletion, kconfigwidgets, kcrash, kdbusaddons,
-  kdesignerplugin, ki18n, kiconthemes, kio, kitemmodels, kwindowsystem, mysql, qttools,
```

Comment:
You should cc the `mysql/mariadb/akonadi/kde` maintainers here to ask about this name change as this breaks `akonadio.override { mysql = foo; }`